front/components/assistant/AssistantPicker.tsx

```diff
@@ -81,7 +82,7 @@ export function AssistantPicker({
           }}
         />
         <DropdownMenuSeparator />
-        <ScrollArea className="border-1 -mb-1 -mt-1 h-[300px]">
+        <ScrollArea className="h-[300px]" hideScrollBar>
```

Comment:
This fixed height will show a blank space if there are not enough items, this should make it dynamic to content 
https://github.com/radix-ui/primitives/issues/2307#issuecomment-2374454843

Reply:
Sweet thx !